ICRCs/ICRC-7/ICRC-7.md

```diff
@@ -160,7 +277,7 @@ type TransferArgs = record {
 };
```

Comment:
`// type: leave open for now`

This should probably be defined similar to ICRC-1 spec:
```
The memo parameter is an arbitrary blob that has no meaning to the ledger. The ledger SHOULD allow memos of at least 32 bytes in length.
```

Reply:
Indeed, this is already in the text below the data structure:

The `memo` parameter is an arbitrary blob that is not interpreted by the ledger.
The ledger SHOULD allow memos of at least 32 bytes in length.
The ledger SHOULD use the `memo` argument for [transaction deduplication](#transaction_deduplication).

Is this what you meant?